website/docs/usage/processing-pipelines.mdx

```diff
@@ -88,11 +88,12 @@ have to call `list()` on it first:
 
 </Infobox>
 
-You can use the `as_tuples` option to pass additional context along with each
-doc when using [`nlp.pipe`](/api/language#pipe). If `as_tuples` is `True`, then
-the input should be a sequence of `(text, context)` tuples and the output will
-be a sequence of `(doc, context)` tuples. For example, you can pass metadata in
-the context and save it in a [custom attribute](#custom-components-attributes):
+You can use the [`nlp.pipe_as_tuples`](/api/language#pipe_as_tuples) method to
+pass additional context along with each doc when using the functionality of
+[`nlp.pipe`](/api/language#pipe). The input should be a sequence of
```

Comment:
The "when using the functionality of `nlp.pipe`" bit of the sentence doesn't quite parse anymore to readers who might not be aware that the same function is still called under the hood.